### PR TITLE
i18n(es): Update `customization.mdx` & `configuration.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- Closes # N/A
- What does this PR change? Give us a brief description.
Updates `customization.mdx` & `configuration.mdx` based on this commit 21b36201aa1e01c8395d0f24b2fa4e32b90550bb & cae82bc6c9d5c1d0cce0ee922ce7aecaf47e72ce

### CODE REVIEW

**Review by Waxer59 — APPROVED** [review]
LGTM ✅

File: docs/src/content/docs/es/reference/configuration.mdx [hunk 1]
```diff
@@ -514,6 +514,13 @@ Establece un delimitador entre el título de la página y el título del sitio w
 Por defecto, cada página tiene un `<title>` de `Título de la página | Título del sitio web`.
 Por ejemplo, esta página es titulada "Referencia de Configuración" y este sitio web es titulado "Starlight", por lo que el `<title>` de esta página es "Referencia de Configuración | Starlight".
 
+### `disable404Route`
+
+**tipo:** `boolean`  
+**por defecto:** `false`
+
+Deshabilita la inyección de la [página 404](https://docs.astro.build/es/core-concepts/astro-pages/#página-de-error-404-personalizada) predeterminada de Starlight. Para usar una ruta `src/pages/404.astro` personalizada en tu proyecto, establece esta opción en `false`.
```
Comment (delucis) [inline docs/src/content/docs/es/reference/configuration.mdx:522]: We realised in #1431 that we made a mistake in documenting this 😅 

This should be `true` here:

```suggestion
Deshabilita la inyección de la [página 404](https://docs.astro.build/es/core-concepts/astro-pages/#página-de-error-404-personalizada) predeterminada de Starlight. Para usar una ruta `src/pages/404.astro` personalizada en tu proyecto, establece esta opción en `true`.
```

**Review by delucis — COMMENTED** [review]
Thanks @dreyfus92! Just one change after we had to fix something in the English version 🙌

**Comment by dreyfus92** [comment]
Thank you both @Waxer59 & @delucis for your reviews 😁

**Review by kevinzunigacuellar — APPROVED** [review]
🥳  Thanks @Waxer59 and @dreyfus92

**Review by delucis — APPROVED** [review]
Let's go, thanks team!

**Pull request merged**